symfony/framework-bundle/3.3/config/services.yaml

```diff
@@ -21,9 +21,8 @@ services:
         # but if a service is unused, it's removed anyway
         exclude: '../src/{Entity,Repository,Tests}'
 
-    # controllers are imported separately to make sure they're public
-    # and have a tag that allows actions to type-hint services
+    # controllers are imported separately to make sure they have the tag
```

Comment:
I'm not sure about the need to mention here that this tag internally makes these services public. Would a comment like this be better?

```
# controllers are imported separately to apply the tag that allows actions to type-hint services
```